tests/functional/common.bash.in

```diff
@@ -386,6 +386,11 @@ function setup_common(){
 
 # Cleanup anything generated by this script
 function cleanup_common(){
+	# kill shim if it is still running
```

Comment:
I don't think so, each tests calls to ```kill``` and ```delete``` if something fails will be reported
after that no cc-shim process should be running